toolbox/deps_update/main.go

```diff
@@ -34,7 +34,8 @@ var (
 const (
 	istioDepsFile         = "istio.deps"
 	prTitlePrefix         = "[DO NOT MERGE] Auto PR to update dependencies of "
-	prBody                = "This PR will be merged automatically once checks are successful."
+	prBody                = "This PR will be merged automatically once checks are successful." +
+		"\r\n```release-note\r\nNONE\r\n```"
```

Comment:
Perhaps define the release-note none as a constant string in CommonUtil.go

Reply:
You are right. I add a param in CreatePullRequest() to handle release-note, leaving it empty will make it "none"